resources/spec.yaml

```diff
@@ -8018,6 +8100,79 @@ definitions:
     type: string
     x-okta-tags:
       - User
+  Authenticator:
+    properties:
+      _links:
+        additionalProperties:
+          type: object
+        readOnly: true
+        type: object
+      created:
+        format: date-time
+        readOnly: true
+        type: string
+      id:
+        readOnly: true
+        type: string
+      key:
+        type: string
+      status:
+        $ref: '#/definitions/AuthenticatorStatus'
+      lastUpdated:
+        format: date-time
+        readOnly: true
+        type: string
+      name:
+        type: string
+      type:
+        $ref: '#/definitions/AuthenticatorType'
+      settings:
+        $ref: '#/definitions/AuthenticatorSettings'
+    type: object
+    x-okta-operations:
+      - alias: activate
+        arguments:
+          - dest: authenticatorId
+            src: id
+        operationId: activateAuthenticator
+      - alias: deactivate
+        arguments:
+          - dest: authenticatorId
+            src: id
+        operationId: deactivateAuthenticator
+    x-okta-tags:
+      - Authenticator
+  AuthenticatorSettings:
+    properties:
+      allowedFor:
+        enum:
```

Comment:
AFAIR, all enums should be separated, i.e. it should look like this:
```
allowedFor:
  $ref: '#definitions/AllowedForEnum'
```
(or any other name for enum) 

I don't remember real reason for this, I guess some SDK requires it

Reply:
Updated.